cdap-app-fabric/src/main/java/io/cdap/cdap/app/metrics/WorkflowMetrics.java

```diff
@@ -46,4 +49,16 @@ public void gauge(String metricName, long value) {
     programMetricsContext.gauge(metricName, value);
     workflowMetricsContext.gauge(metricName, value);
   }
+
+  @Override
+  public Metrics child(Map<String, String> tags) {
+    return new WorkflowMetrics(programMetricsContext.childContext(tags), workflowMetricsContext.childContext(tags));
+  }
+
+  @Override
+  public Map<String, String> getTags() {
+    Map<String, String> result = new HashMap<>();
+    result.putAll(programMetricsContext.getTags());
+    return null;
```

Comment:
needs to be updated.

Reply:
This class is actually seems unused. It was introduced here - https://issues.cask.co/browse/CDAP-5450 in order to emit metrics in two different contexts for program running under workflow, one for program itself and another for workflow. However turned out that caused duplicate counting at namespace/app level - https://issues.cask.co/browse/CDAP-5656. So in https://issues.cask.co/browse/CDAP-5656 instead of emitting metrics at 2 context we added additional worklow related tags to the program metrics context itself.  This made `WorklowMetrics` class redundant.